docs/user-guide/widgets.md

```diff
@@ -37,6 +37,14 @@ Selecting a *Layer* or *Layers*, the following *Chart* options is presented to u
 
 <img src="../img/widgets/chart-options.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
+From the top toolbar of this panel the user is allowed to:
```

Comment:
@ElenaGallo if you want to specify this at table widget level and chart level why you shouldn't do that also for counter widgets?